find-bug.py

```diff
@@ -178,6 +178,8 @@
                     help='If given, ensure that the computed error message occurs in this log.')
 parser.add_argument('-y', '--yes', '--assume-yes', dest='yes', action='store_true',
                     help='Automatic yes to prompts. Assume "yes" as answer to all prompts and run non-interactively.')
+parser.add_argument('--no-color', dest='coloron', action='store_const', const=False, default=True,
```

Comment:
Slight preference for
```suggestion
parser.add_argument('--no-color', dest='color_on', action='store_const', const=False, default=True,
```
(here and elsewhere)

Reply:
Done